### PR TITLE
[core:admin] Exp-435 Add audit logs page in the admin panel settings

### PR DESCRIPTION
**Pull request by madhurisandbhor**
### What does it do?

Added audit logs page in admin panel settings

### How to test it?

Go to settings > under "Administration Panel" section user can find "Audit Logs" link

### CODE REVIEW

File: packages/core/admin/admin/src/pages/SettingsPage/pages/AuditLogs/ListView/tests/index.test.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  </ThemeProvider>
+);
+
+describe('ADMIN | Pages | AUDIT LOGS | ListView', () => {
+  it('renders and matches the snapshot', () => {
+    const {
+      container: { firstChild },
+    } = render(App);
+
+    expect(firstChild).toMatchInlineSnapshot(`
```
Comment (gu-stav) [inline packages/core/admin/admin/src/pages/SettingsPage/pages/AuditLogs/ListView/tests/index.test.js:27]: What do you think about removing the snapshot test and replace it with more granular tests (e.g. check if the title is properly rendered)? In the end this page will become big (markup-wise) and I would find it more useful to have smaller tests ...

Comment (madhurisandbhor) [reply packages/core/admin/admin/src/pages/SettingsPage/pages/AuditLogs/ListView/tests/index.test.js:27]: sure, yes I also think so, that's a better way of testing and snapshots should be for very specific formats. But I just tried to follow other pages.

**Review by remidej — COMMENTED** [review]
This works nicely 🙂 

For the tests I agree with Gustav it sounds like a good idea to try something new here

**Review by remidej — APPROVED** [review]

**Comment by Svirsaq** [comment]
What the fuck??

**Review by joshuaellis — APPROVED** [review]

**Pull request merged**